pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/impl/invertedindex/RealtimeLuceneTextIndex.java

```diff
@@ -78,7 +78,7 @@ public RealtimeLuceneTextIndex(String column, File segmentIndexDir, String segme
       // for realtime
       _indexCreator =
           new LuceneTextIndexCreator(column, new File(segmentIndexDir.getAbsolutePath() + "/" + segmentName),
-              false /* commitOnClose */, true, null, config);
+              false /* commitOnClose */, true, null, null, config);
```

Comment:
Seems like you are passing in `null` as `consumerDir` here? Do we have a test covering this path right now? Won't it cause NPE?

Reply:
We do - it's covered in `LuceneMutableTextIndexTest`. `consumerDir` isn't used for the realtime index, and is only used by the LuceneTextIndexCreator initialized during conversion of the realtime segment.